File: Data Structures & Algorithms/anagram-groups/submission-0.py
class Solution:
    def groupAnagrams(self, strs: List[str]) -> List[List[str]]:
         my_dict = defaultdict(list)
         for i in strs:
            my_dict["".join(sorted(i))].append(i)
         ans = []
         for i in my_dict:
            ans.append(my_dict[i])
         return ans

File: Data Structures & Algorithms/buy-and-sell-crypto/submission-0.cpp
class Solution {
public:
    int maxProfit(vector<int>& prices) {

     int n = prices.size();
     int l = 0,r = 1;
     int ans =0;
     for( ;r<n;r++)

     {
        if(prices[l] < prices[r])
        {
            int p = prices[r]-prices[l];
            ans = max ( ans , p);
        }
        else{
            l = r;
        }
        // cout << l << " : " << r << endl;
     }
     return ans;   
    }
};
